assets/js/components/KeyMetrics/ConnectGA4CTAWidget.test.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * ConnectGA4CTAWidget component tests.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import ConnectGA4CTAWidget from './ConnectGA4CTAWidget';
+import {
+	CORE_USER,
+	KM_ANALYTICS_ENGAGED_TRAFFIC_SOURCE,
+	KM_ANALYTICS_LOYAL_VISITORS,
+	KM_ANALYTICS_NEW_VISITORS,
+	KM_ANALYTICS_TOP_TRAFFIC_SOURCE,
+	KM_SEARCH_CONSOLE_POPULAR_KEYWORDS,
+	KM_ANALYTICS_ADSENSE_TOP_EARNING_CONTENT,
+} from '../../googlesitekit/datastore/user/constants';
+import {
+	render,
+	createTestRegistry,
+	provideModules,
+	provideKeyMetrics,
+	provideUserAuthentication,
+	provideUserCapabilities,
+} from '../../../../tests/js/test-utils';
+import { CORE_WIDGETS } from '../../googlesitekit/widgets/datastore/constants';
+import { AREA_MAIN_DASHBOARD_KEY_METRICS_PRIMARY } from '../../googlesitekit/widgets/default-areas';
+import { CONTEXT_MAIN_DASHBOARD_KEY_METRICS } from '../../googlesitekit/widgets/default-contexts';
+
+describe( 'ConnectGA4CTAWidget', () => {
+	let registry;
+
+	const DISMISSED_ITEM_KEY = 'key-metrics-connect-ga4-cta-widget';
+
+	const Widget = ( { children } ) => <div>{ children }</div>;
+	const WidgetNull = () => <div>NULL</div>;
```

Comment:
Why override these? I see that this has been done in a few other places but don't see why we'd not use the normal components. We're already doing that in the story, so we should be able to do the same here https://github.com/google/site-kit-wp/blob/ea139720edaec9027893f64e02c9cd5c775ed28b/assets/js/components/KeyMetrics/ConnectGA4CTAWidget.stories.js#L37-L39

Reply:
Good idea. I've used this in the tests too. Thank you!